variables.tf

```diff
@@ -48,6 +48,12 @@ variable "shared_vpc" {
   default     = ""
 }
 
+variable "enable_shared_vpc_host_project" {
+  description = "If this project is a shared VPC host project. If true, you must set shared_vpc variable. Default is false for compatibility"
```

Comment:
```suggestion
  description = "If this project is a shared VPC host project. If true, you must set *not* shared_vpc variable. Default is false."
```

Reply:
@morgante actually, no... shared_vpc is the name of the host vpc, this is a boolean.